kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/Main.java

```diff
@@ -96,9 +98,11 @@
     jc.addCommand("schema", new SchemaCommand(console));
     jc.addCommand("info", new InfoCommand(console));
     jc.addCommand("show", new ShowRecordsCommand(console));
+    jc.addCommand("obj-schema", new ObjectSchemaCommand(console));
     jc.addCommand("csv-schema", new CSVSchemaCommand(console));
     jc.addCommand("csv-import", new CSVImportCommand(console));
-    jc.addCommand("obj-schema", new ObjectSchemaCommand(console));
+    jc.addCommand("json-schema", new JSONSchemaCommand(console));
+    jc.addCommand("json-import", new JSONImportCommand(console));
```

Comment:
Hrm, you're right that we do need a balance between the number of individual commands we have to expose versus the number of options per command. I'm not sure exactly where that balance is. I'll keep thinking of a better name for informat-import and think more on if there's an elegant way to consolidate the three commands at all.